requirements/edx/base.txt

```diff
@@ -146,7 +146,7 @@ bok-choy==0.5.3
 chrono==1.0.2
 coverage==4.0.2
 ddt==0.8.0
-diff-cover==0.9.0
+git+git://github.com/bjacobel/diff-cover.git@bjacobel/eslint#egg=diff-cover
```

Comment:
It was merged a few hours ago, and the maintainer told me he'd be releasing a new version to pypi tonight.